### ISSUE TITLE
Change the debug sidebar icon

### ISSUE THREAD

**Issue by danebou** [microsoft/vscode#79218]
The new redesign has been looking great. Although it might just be personal preference but the new design for the debug side bar icon is a bit interesting: 

![image](https://user-images.githubusercontent.com/5658563/63102840-dba43380-bf49-11e9-8c95-6df6e8c98a5a.png)

It looks so sharp, that it looks like I'm going to impale my cursor on its legs, and it gives me a bit of anxiety!

Put more concretely, all the icons are smooth and have rounded edges throughout VsCode. This icon is a bit of an outlier being so "sharp". 

A few suggestions are "nubs" for legs:
![image](https://user-images.githubusercontent.com/5658563/63102816-d21acb80-bf49-11e9-94cc-e5d471ea2782.png)

Additionally the "x" isn't as clear its a debug as the old "not" symbol:
![image](https://user-images.githubusercontent.com/5658563/63102949-202fcf00-bf4a-11e9-9d6e-4d244fa7ea6f.png)

So further iterating:

![image](https://user-images.githubusercontent.com/5658563/63103093-700e9600-bf4a-11e9-8c0b-43ed98ee46c3.png)

**Comment by isidorn** [comment]
Assigning first to @misolori to get his feedback

**Comment by misolori** [comment]
Thanks for the suggestion! We did iterate on this icon numerous times before we landed on the current one as we couldn't find the right fit for it:

<img width="782" alt="image" src="https://user-images.githubusercontent.com/35271042/63104037-7a4a8780-bf7e-11e9-8c2e-8b09e2c15d4b.png">

While I appreciate your suggestion, I think that makes it a bit more complicated and harder to recognize. If we do think about revisiting this icon I'll be sure and re-open this issue. Thanks!
